book/doctrine.rst

```diff
@@ -134,6 +134,12 @@ for you:
     There's no way to configure these defaults inside Doctrine, as it tries to be
     as agnostic as possible in terms of environment configuration. One way to solve
     this problem is to configure server-level defaults.
+    
+.. caution::
+
+    If you are using MySQL, its `utf8` character set has some shortcomings 
+    which may cause problems. Prefer the `utf8mb4` character set instead, if 
+    your version supports it.
```

Comment:
Hmm, I guess I *did* want to have a caution-block inside the sidebar-block, but offhand I'm not sure if that's a thing that is done elsewhere in the documentation.